cat/CaT-service.yaml

```diff
@@ -2530,20 +2534,18 @@ paths:
       security:
         - OAuth2:
             - buyer
-      summary: Create further info request
+      summary: Create a message for an event
       
-  /tenders/projects/{proc-id}/events/{event-id}/responses/suppliers/{supplier-id}/furtherInfo:
+  /tenders/projects/{proc-id}/events/{event-id}/messages/{message-id}:
     parameters:
       - $ref: '#/components/parameters/ProcurementParam'
       - $ref: '#/components/parameters/EventParam'
-      - $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Params.yaml#/components/parameters/SupplierParam'
-
-
+      - $ref: '#/components/parameters/MessageParam'
     get:
-      description: 'Get a request for further information (includes free text and any files}'
+      description: 'Gets a list of messages for the user relating to this event}'
```

Comment:
Looks like a copy-paste from get all messages (think this is just getting a single message)

Reply:
Made fix in latest version, thanks